pkg/ui/output.go

```diff
@@ -14,20 +15,26 @@ import (
 	"golang.org/x/term"
 )
 
+var removeAnsi = regexp.MustCompile(`(?i)\\\\u00[1-9]b[[0-9;]*[mGKHF]`)
+var emojiStatus = []string{color.GreenString("✓"), "📋", color.RedString("❌"), "⚠️", "⌛"}
+
 // ColorizedOutput outputs a colored message directly to the terminal.
 // The remaining arguments should be interpolations for the format string.
 func ColorizedOutput(c *color.Color, msg string, values ...interface{}) {
 	if viper.GetBool("enable-console-log") {
 		// Print output to log
-		if logMsg := strings.ReplaceAll(msg, "\n", ""); logMsg != "" {
-			log.Info().Msgf(logMsg, values...)
+		if logMsg := strip(fmt.Sprintf(msg, values...)); logMsg != "" {
```

Comment:
[non blocking] Can you explain the reason we need to strip emojis in the console (maybe good to comment it)?

Also, we could consider using https://github.com/acarl005/stripansi/blob/5a71ef0e047df0427e87a79f27009029921f1f9b/stripansi.go#L7 or https://github.com/chalk/ansi-regex/blob/02fa893d619d3da85411acc8fd4e2eea0e95a9d9/index.js#L3

Reply:
I tried them they didn't work cleanly for some reason